src/main/java/htsjdk/samtools/util/SnappyLoader.java

```diff
@@ -24,28 +24,20 @@
 package htsjdk.samtools.util;
 
 import htsjdk.samtools.SAMException;
-import org.xerial.snappy.LoadSnappy;
 import org.xerial.snappy.SnappyInputStream;
+import org.xerial.snappy.SnappyOutputStream;
 
+import java.io.ByteArrayOutputStream;
 import java.io.InputStream;
 import java.io.OutputStream;
-import java.lang.reflect.Constructor;
 
 /**
  * If Snappy is available, obtain single-arg ctors for SnappyInputStream and SnappyOutputStream.
  */
 public class SnappyLoader {
     private static final int SNAPPY_BLOCK_SIZE = 32768;  // keep this as small as can be without hurting compression ratio.
-    private final Constructor<InputStream> SnappyInputStreamCtor;
-    private final Constructor<OutputStream> SnappyOutputStreamCtor;
     public final boolean SnappyAvailable;
 
-    // Force Snappy-java code to be loaded into executable jars.
-    private final SnappyInputStream ignoreMe = null;
```

Comment:
@yfarjoun - this used to be the case when a) downstream projects tried to be "smart" about only including necessary classes in their fat jars instead of just including all dependencies and b) the code in `SnappyLoader` didn't explicitly reference any classes from the snappy-java library.  With this latest release it works to just attempt to use the snappy-java classes, whereas (I think) in the old release we were using, you would get `ClassNotFoundError`s because static initializers would fail if the native libraries could not be loaded.